Mathlib/MeasureTheory/Function/L1Space/HasFiniteIntegral.lean

```diff
@@ -177,6 +178,14 @@ theorem hasFiniteIntegral_const [IsFiniteMeasure μ] (c : β) :
     HasFiniteIntegral (fun _ : α => c) μ :=
   hasFiniteIntegral_const_iff.2 <| .inr ‹_›
 
+theorem HasFiniteIntegral.of_mem_Icc_enorm [IsFiniteMeasure μ]
```

Comment:
The name feels a bit off to me since `enorm` doesn't appear anywhere. Maybe this?
```suggestion
theorem HasFiniteIntegral.of_mem_Icc_of_ne_top [IsFiniteMeasure μ]
```

Reply:
Good point - and thanks for the suggestion, which I like!